### ISSUE TITLE
customeditor를 사용해서 select 의 option을 추가하면 text가 아니라 value가 표시되는 현상

### ISSUE THREAD

**Issue by higamza** [nhn/tui.grid#704]
**Summary**
1. customEditor를 사용해서 select를 동적으로 생성하고 option도 동적으로 추가했습니다.
그런데 그렇게 생성된 select의 값을 선택하고 다른 셀로 이동하면
select 박스에 option의 text 값이 아닌 value가 표시되요
예를 들어 <option value="FRUIT">과일</option> 일 경우
'과일'을 선택하면 셀에 '과일' 이라는 텍스트가 표시되는게 아니라 'FRUIT' 라는 텍스트가 표시됩니다.

2. editor : {type :'select'... 로 옵션을 준 셀에서는 onAfterChange 이벤트가 발생하지 않는데
select 박스의 값을 선택하는건 셀 값의 변경으로 인식되지 않는건가요?

**Screenshots**
![image](https://user-images.githubusercontent.com/4151146/65833070-e43ca780-e306-11e9-9053-08358a3ec7a0.png)

**Version**
grid 4.5.2

**Additional context**
formatter, editor 로 생성한 select박스는 제대로 선택한 옵션의 text가 표시되는데
customEditor로 생성한 select만 value가 표시되네요

아래는 전체 소스 링크입니다.
https://github.com/higamza/gridtest/blob/master/customselect1

**Comment by soyun9495** [comment]
혹시 해결하셨나요?ㅠㅠ 해결하셨다면 공유 부탁드립니다

**Comment by higamza** [comment]
그 당시 프로젝트에서 쓰려고 이래저래 시도하던건데 ㅎㅎㅎ님 메일받고 생각나서 심심해서 수정해봤어요아래 getValue() 함수 부분처럼 수정하니까 되네요근데 나중에 그리드의 값을 가져오려고 할때 value값이 아니라 label 값을 가져올 것도 같고 흠..그건 테스트를 안해봐서 ㅎㅎ;;<!doctype html><html> <head> <meta charset="UTF-8"><link rel="stylesheet" href="https://uicdn.toast.com/tui-grid/latest/tui-grid.css" /><script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script><script src="https://uicdn.toast.com/tui-grid/latest/tui-grid.js"></script> <title>Document</title><script type="text/javascript" charset="utf-8">  var firstval = [{label: '과일', value: 'FRUIT'},{label: '채소', value: 'VEGETABLE'}];  var secondval = [{type: 'FRUIT', label: '사과', value: 'APPLE'}, {type: 'FRUIT', label: '오렌지', value: 'ORANGE'},{type: 'VEGETABLE', label: '상추', value: 'LETTUCE'}, {type: 'VEGETABLE', label: '마늘', value: 'GALIC'}];  var Grid = tui.Grid;$(function(){	class CustomEditorFirstSelect {  constructor(props) {  const el = document.createElement('select');      let fn = firstval.length;  let fitem;  for(let i=0; i < fn; i++){  fitem = firstval[i];  el.append(new Option(fitem.label, fitem.value));    }  this.el = el;  }  getElement() {  return this.el;  }  getValue() {  return this.el[this.el.selectedIndex].label;  }  mounted() {  //this.el.select();  }	}	const gridData = [    {      c1: '100013',      c2: '',      c3: 1,	  c4: 'test1'    },    {      c1: '201212',      c2: '',      c3: 2,	  c4: 'test2'    }  ];  const grid = new tui.Grid({  el: document.getElementById('grid'),  editingEvent: 'click',  scrollX: false,  scrollY: false,  data: gridData,  columns: [  {  header: 'Name',  name: 'c1'  },  {  header: 'first',  name: 'c2',  editor: {  type: CustomEditorFirstSelect  }  },  {  header: 'normalselect',  name: 'c4',  formatter: 'listItemText',  editor : {  type: 'select',  options : {  listItems: [  {text: '테스트1', value: 'test1'},  {text: '테스트2', value: 'test2'}  ]  }  },  onAfterChange(ev){  alert('1');  }  }  ]  });});  </script></head><body> <p>grid test</p>  <div id="grid"></div></body></html>회사 단톡방 스트레스 치료는 '네이트온'
------ Original Message ------
Date: Friday, Jul 2, 2021 11:00:01 AM
